bait-files/src/main/webapp/review.js

```diff
@@ -44,28 +45,40 @@ function getRequest() {
   fetch('/request' + queryString)
       .then(response => response.json())
       .then((request) => {
-          addRequestToPage(request);
+        addRequestToPage(request);
+
+        let blobKeyString = request.image;
```

Comment:
can this go into a separate function that gets called here with request.image as the parameter?